src/Sylius/Component/Core/spec/Model/ProductSpec.php

```diff
@@ -67,6 +68,19 @@ function its_taxons_are_mutable(Collection $taxons)
         $this->getTaxons()->shouldReturn($taxons);
     }
 
+    function its_taxons_should_be_returned_by_taxonomy(
+        TaxonInterface $taxon,
+        TaxonInterface $rootTaxon
+    ) {
+        $rootTaxon->getName()->willReturn('Brand');
+        $taxon->getRoot()->willReturn($rootTaxon);
+        $this->setTaxons(new ArrayCollection([$taxon->getWrappedObject()]));
+
+        $returnedTaxons = $this->getTaxons('brand');
+        $returnedTaxons->shouldHaveType('Doctrine\Common\Collections\Collection');
+        $returnedTaxons->count()->shouldReturn(1);
```

Comment:
`$this->getTaxons('brand')->shouldReturn(new ArrayCollection([$taxon->getWrappedObject()]));`